docs/reference/objects/package/product.md

```diff
@@ -16,22 +17,31 @@ Represents a [product]({% link docs/reference/objects/product/index.md %}) that'
 #### Attributes
 
 ## `package_product.id`
+
 {: .d-inline-block }
 string
 {: .label .fs-1 }
 
 The unique id of the package product.
 
 ## `package_product.product`
+
 {: .d-inline-block }
 [Product]({% link docs/reference/objects/product/index.md %})
 {: .label .fs-1 }
 
 The specific product that's associated to the Package Product.
 
 ## `package_product.variants`
+
 {: .d-inline-block }
 Array of [Variant]({% link docs/reference/objects/product/variant/index.md %})
 {: .label .fs-1 }
 
-The variants of the `package_product.product` that the creator has included in the package and are eligible for booking by the customer.
```

Comment:
Looks like this description has been removed from the variants section

Reply:
If its to be replaced altogether, could we add the deprecated label instead? 
e.g. 
![CleanShot 2024-04-23 at 15 26 23](https://github.com/easolhq/easolhq.github.io/assets/48216064/0fa515e1-e179-4b42-b186-691e389d9777)